TTHAnalysis/python/plotter/mcAnalysis.py

```diff
@@ -8,8 +8,8 @@
 
 ## These must be defined as standalone functions, to allow runing them in parallel
 def _runYields(args):
-    key,tty,cuts,noEntryLine,fsplit = args
-    return (key, tty.getYields(cuts,noEntryLine=noEntryLine,fsplit=fsplit))
+    key,tty,cuts,noEntryLine,fsplit,addCut = args
+    return (key, tty.getYields(cuts,noEntryLine=noEntryLine,fsplit=fsplit,addCut=addCut))
```

Comment:
It's not nice to change the interface of such important functions if not strictly needed.
Why do we need this addCut? I think we should print the exact thing that goes in the plot at all times, otherwise it gets quite confusing. We can always create a different line in the plot file to have different binning.
If I'm not missing the reason, I propose to remove addCut everywhere and use the same binning and selection as the plot.